<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maze</class>
 <widget class="QMainWindow" name="Maze">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maze</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
selection-color: rgb(46, 194, 126);;
background-color:  rgb(41, 41, 41)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color:  rgb(54, 54, 60);
background-color: rgb(30, 32, 25);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuUpload">
     <property name="title">
      <string>Upload...</string>
     </property>
     <addaction name="actionMaze"/>
     <addaction name="actionCave"/>
    </widget>
    <addaction name="menuUpload"/>
    <addaction name="actionGenerate_cave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuAlgorithm">
     <property name="title">
      <string>Cave algorithm</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionStep_by_step"/>
     <addaction name="actionAutomode"/>
    </widget>
    <addaction name="actionSolve_current_maze"/>
    <addaction name="menuAlgorithm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 32, 25);</string>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionGenerate_cave">
   <property name="text">
    <string>Generate maze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSolve_current_maze">
   <property name="text">
    <string>Solve maze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionMaze">
   <property name="text">
    <string>Maze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionCave">
   <property name="text">
    <string>Cave</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionStep_by_step">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Step-by-step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAutomode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
